CMakeLists.txt

```diff
@@ -64,7 +64,21 @@ endif()
 
 ################################################################################
 
-find_package(mdspan REQUIRED)
+find_package(mdspan QUIET)
+if (NOT mdspan_FOUND)
+  message(STATUS "No installed mdspan found, fetching from Github")
+  include(FetchContent)
+  FetchContent_Declare(
+    mdspan
+    GIT_REPOSITORY https://github.com/kokkos/mdspan.git
+    GIT_TAG        5694f21c39f3b948d06a0c63b9c219bf802e28a8
```

Comment:
The implementation of mdspan has been changing lately to track P0009 changes, so we tend to like fetching from `HEAD` of the `stable` branch.  Is there a way to do that, or do we have to pick a specific commit?

Reply:
Yes, `GIT_TAG` can be either a commit hash, a branch name or a tag name. The [`ExternalProject`](https://cmake.org/cmake/help/latest/module/ExternalProject.html) documentation provides more details on the possible parameters

Reply:
@upsj Could you actually please change this to fetch from HEAD of the stable branch?  Thanks! : - )